posts/2021-02-19-mpcontextpropagation-requesttiming-21003-beta.adoc

```diff
@@ -0,0 +1,332 @@
+
+The main change visible to developers in Jakarta EE is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have a number of new API Release Candidates to join the expanding library of supported Jakarta packages.
+
+This Open Liberty beta introduces the following Jakarta EE 9 feature which now possess it's all-new Jakarta EE 9 package name:
+
+* <<mail, JakartaMail (`mail-2.0`)>>
+
+This feature joins the Jakarta EE 9 features in link:{url-prefix}/blog/2021/01/26/ee9-messaging-security-21002-beta.html#jakarta[Open Liberty 21.0.0.2-beta Jakarta functions].
+
+'''
+
+[#mail]
+=== JakartaMail
+
+The Java EE framework has been migrated to the open source Eclipse Jakarta EE Project. As part of this migration JavaMail version 1.6 has been migrated to JakartaMail 2.0. The API package names for the classes previously found under the javax.mail have been migrated to jakarta.mail. 
+
+The Jakarta mail API as described by the Jakarta Mail FAQ “The Jakarta Mail API is a set of abstract APIs that model a mail system. (Jakarta Mail was previously known as JavaMail.) The API provides a platform independent and protocol independent framework to build Java technology based email client applications. The Jakarta Mail API provides facilities for reading and sending email. Service providers implement particular protocols. Several service providers are included with the Jakarta Mail API package; others are available separately. The Jakarta Mail API is implemented as a Java optional package that can be used on JDK 1.4 and later on any operating system. The Jakarta Mail API is also a required part of the Jakarta EE Platform and the Java Platform, Enterprise Edition (Java EE).”
+
+Configuring mail sessions works basically the same as with the Liberty Feature JavaMail-1.5 and JavaMail-1.6. They can be configured using the API, or through the server.xml 
+
+Below is an example of a SMTP Mail session configured through the server.xml:
+
+[source, xml]
+----
+<featureManager>
+  <feature>mail-2.0</feature>
+  <feature>jndi-1.0</feature>
+</featureManager>
+
+<mailSession>
+      <mailSessionID>testSMTPMailSession</mailSessionID>
+      <jndiName>TestingApp/SMTPMailSessionServlet/testSMTPMailSession</jndiName>
+      <description>mailSession for testing SMTP protocol</description>
+      <transportProtocol>smtp</transportProtocol>
+      <host>localhost</host>
+      <user>somuser@someemailserver.com</user>
+      <password>usersPassword</password>
+      <from>someuser@someemailserver.com</from>
+      <property name="mail.smtp.host" value="localhost" \>
+      <property name="mail.smtp.port" value="3025" \>
+  </mailSession> 
+----
+
+Find out more:
+
+* link:https://eclipse-ee4j.github.io/mail/[Jakarta mail]
+* link:https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_admin_javamail.html[Administering on liberty substitute JavaMail-1.5 with mail-2.0]
```

Comment:
I think we agreed to remove this link ... but did you want to ask the developer first?

Reply:
Waiting for the dev to get back to me. Will check with him again today.

Reply:
Done